overlord/servicestate/quota_handlers.go

```diff
@@ -912,6 +912,9 @@ func validateSnapServicesForAddingToGroup(st *state.State, services []string, gr
 		if parentGroup == nil || !strutil.ListContains(parentGroup.Snaps, snap) {
 			return fmt.Errorf("cannot use snap service %q: the snap %q must be in a direct parent group of group %q", service, snap, group)
```

Comment:
I didn't notice this before but why "use" instead of "add" ?, same for similar errors in this code file

Reply:
Good point, I think it was my danish semantics getting mixed up with english, corrected these to be more semantically correct.